app/forms/declaration/DispatchLocation.scala

```diff
@@ -48,4 +49,10 @@ object DispatchLocation extends DeclarationPage {
     val OutsideEU = "EX"
     val SpecialFiscalTerritory = "CO"
   }
+
+  override def defineTariffContentKeys(decType: DeclarationType): Seq[String] =
+    decType match {
+      case CLEARANCE => Seq("tariff.clearance.dispatchLocation")
+      case _         => Seq("tariff.common.dispatchLocation", "tariff.common.dispatchLocation")
+    }
```

Comment:
I'd change this method's name to sth a bit "simpler", like just `getTariffContent`. 

But more importantly, I'm considering a different return type. `Seq[String]` is very generic and you can't actually provide correct return values without going through several other files to see how it is being used.
I'd suggest creating a `case class` that would hold the keys in more descriptive way, or I'd consider returning ready text (extracted from `Messages`).